master/buildbot/data/builds.py

```diff
@@ -20,6 +20,14 @@
 
 class Db2DataMixin(object):
 
+    def injectProperties(self, data, props, filters=None):
```

Comment:
if this method is not expected to be used outside of this class, `_injectProperties` would be a better name